sidebars.js

```diff
@@ -273,6 +252,32 @@ module.exports = {
       ],
     },
   ],
+  nodejsSidebar: [
+    {
+      type: 'doc',
+      id: 'node/reference/README',
+      label: 'Overview',
+    },
+    'node/getting-started',
+    'node/hello-world',
+    {
+      type: 'category',
+      label: 'API Reference',
+      items: [
+        { type: 'doc', id: 'node/meta-package', label: 'temporalio' },
```

Comment:
This won't be necessary if we don't use a submodule I think -

Reply:
We still will need this.